daemon/postprocess/PrePostProcessor.h

```diff
@@ -42,8 +42,10 @@ class PrePostProcessor : public Thread, public Observer
 	virtual void Update(Subject* caller, void* aspect) { DownloadQueueUpdate(aspect); }
 
 private:
-	int m_queuedJobs = 0;
+	std::atomic_uint m_queuedJobs{0};
```

Comment:
I use `int` whenever possible. Unsigned types considered dangerous.

Please replace with `std::atomic_int`. There are also few other places with `std::atomic_uint` in other units which should be replaced too. Thanks.

Reply:
changed